steer_cube.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash -l
+
+
```

Comment:
TODO: add note "# this job steering file was developed for Pawsey's Magnus computer system. See https://pawsey.org.au/systems/magnus/"